registry/src/proto.rs

```diff
@@ -1,3 +1,3 @@
 // (c) Copyright 2023 Helsing GmbH. All rights reserved.
 
-buffrs::include!();
+::buffrs::include!();
```

Comment:
why is this needed?

Reply:
I had an error if I did not include that:

```
error[E0433]: failed to resolve: could not find `include` in `buffrs`
 --> registry/src/proto.rs:3:9
  |
3 | buffrs::include!();
  |         ^^^^^^^ could not find `include` in `buffrs`

error[E0659]: `buffrs` is ambiguous
  --> registry/src/proto.rs:3:1
   |
3  | buffrs::include!();
   | ^^^^^^ ambiguous name
   |
   = note: ambiguous because of a conflict between a macro-expanded name and a less macro-expanded name from outer scope during import or macro resolution
   = note: `buffrs` could refer to a crate passed with `--extern`
   = help: use `::buffrs` to refer to this crate unambiguously
note: `buffrs` could also refer to the module defined here
  --> /Users/patrick.elsen/Projects/helsing/buffrs/target/debug/build/buffrs-registry-b5cc01ff6df2be9c/out/buffrs.rs:1:1
   |
1  | / pub mod buffrs {
2  | |     pub mod dependency {
3  | |         include!("buffrs.dependency.rs");
4  | |     }
...  |
16 | |     }
17 | | }
   | |_^
   = help: use `self::buffrs` to refer to this module unambiguously
```

I think the issue is that there are two `buffrs`: the crate `buffrs` and the module `buffrs` which is included here.

Perhaps we could rename the generated module to something like `buffers_gen` instead.